packages/plugin-express/types/bugsnag-express.d.ts

```diff
@@ -0,0 +1,3 @@
+import * as Bugsnag from "@bugsnag/core";
```

Comment:
are wildcard imports conventional for JS? We discourage this style in our Java repos

Reply:
This is typescript, and tslint doesn't seem to complain about it – but since I'm only using the `IPlugin` type from this file I will change it to be `import { IPlugin } from "@bugsnag/core";`